PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+# Description
+
+Please include a summary of the changes and the related issue.
+
+Fixes # (issue)
+
+## Type of change
+
+Please select all options that apply to this change.
+
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] Documentation update
+
+# Checklist:
+
+Please select all options that was done as part of this change.
```

Comment:
Should this be `Please complete all steps before opening a pull request.` and then a numbered list?

Reply:
If you don't want to go the numbered list route, this still needs rewording. If the steps are mandatory, they should not be referred to as options, and it should read "were done" instead of "was done". I'd suggest:

> Confirm you have completed the following steps:

Reply:
updated

Reply:
however the only time not all steps are mandatory is during document update but I guess thats ok